bindata/network/ovn-kubernetes/managed/008-route.yaml

```diff
@@ -15,3 +15,6 @@ spec:
   to:
     kind: Service
     name: ovnkube-master
+{{ if .OVNSbDbRouteHost }}
```

Comment:
Presumably we should also avoid creating this Route at all if it's not needed?

Reply:
Absolutely, missed that